### ISSUE TITLE
tC appears to not always force the download of the original language version used during the initial work in tN 

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/translationCore#7140]
3.0.1
I have one Mac (A) with the latest English and Greek resources, and one (B) with those in the release.
I imported a NT usfm project on machine B and gave it a new unique identifier
I performed a couple of tN checks
I uploaded the project to DCS
I imported the project from DCS to machine A
I loaded it in tN and did not receive the message that a previous version of the original language needs to be loaded.

**Comment by da1nerd** [comment]
@cckozie let me reword this to make sure I understand.
If you import a project, you want to make sure that tC always uses the same version of the language indicated in the project. And download it if necessary.

**Comment by cckozie** [comment]
@da1nerd - tC captures the version of the original language being used when a user opens the project in tN for the first time. tC is designed to then continue to use that same version of the original language for all work in that project's tN. When the user later chooses to open the project in tN, tC checks the manifest to what version was previously used, and then will load that version. If for some reason that version of the original language is not in the resources directory, it prompts the user to download it or cancel the loading of the project in tN. I've not encountered a problem with this except in the scenario I outlined in this issue.
I hope that is clear:)

**Comment by da1nerd** [comment]
@cckozie it appears this was fixed along with my changes for https://github.com/unfoldingword/translationcore/issues/7121

**Comment by gayathrihayden** [comment]
Fix tested in tC 3.0.2 (ccf6be6) and works fine. 
Tested as part of [#7121](https://github.com/unfoldingWord/translationCore/issues/7121)
